examples/webvr_cubes.html

```diff
@@ -60,8 +60,11 @@
 				scene = new THREE.Scene();
 				scene.background = new THREE.Color( 0x505050 );
 
+				var dolly = new THREE.Group();
+				dolly.position.set(0, 1.6, 0);
```

Comment:
Shouldn't this be set with reference frame or WebVR's standing matrix?

Reply:
True, it should be handled on the WebVR / WebXR side but couldn't find a way to get consistent behavior across all API flavors. Specially with the m73 changes. I can remove to keep this simpler.